main.go

```diff
@@ -100,8 +104,10 @@ func main() {
 				grouping = "USAGE_TYPE"
 			}
 
-			firstResultsCosts := GetCosts(svc, firstMonthStart, firstMonthEnd, costMetric, grouping, serviceFilter)
-			secondResultsCosts := GetCosts(svc, secondMonthStart, secondMonthEnd, costMetric, grouping, serviceFilter)
+			tagFilters := c.StringSlice("tag")
```

Comment:
I couldn't figure out how to specify a destination for a `StringSliceFlag`, since the data type is technically a `cli.StringSlice` I think. But this approach to get the specified values seems to work.